src/Twig/Extension/NumberExtension.php

```diff
@@ -51,7 +51,7 @@ public function getFilters(): array
     public function getFunctions(): array
     {
         return [
-            new TwigFunction('defaultCurrency', 'Cake\I18n\Number::defaultCurrency'),
+            new TwigFunction('defaultCurrency', 'Cake\I18n\Number::getDefaultCurrency'),
             new TwigFunction('number_formatter', 'Cake\I18n\Number::formatter'),
```

Comment:
The number formatter was present as a filter as well as function?

Reply:
It's format`ter` vs `format`. I don't know who wants to build a formatter in a twig script though.

Reply:
Oops, I missed the difference :stuck_out_tongue:. But yeah building a number formatter in twig template makes no sense. I would suggest deprecating that too.

Reply:
Added deprecation warning.